docs/quick-start/vote-contract/create-dapp.md

```diff
@@ -70,6 +70,10 @@ useEffect(() => {
 }, [provider]);
 ```
 
+:::tip
+ℹ️ Note: You are to replace the address placeholder with your deployed voting contract address from "Deploy Voting dApp Smart Contract"!
```

Comment:
This involves line 61? It would be better if you highlight the code where the user needs to change the address at. In this tip section, also specify the line number.